src/scripts/lofreq2_somatic.py

```diff
@@ -563,7 +564,7 @@ def run(self):
 
         for (k, v) in [(x, self.__getattribute__(x)) for x in dir(self)
                        if not x.startswith('_')]:
-            if callable(v):
+            if isinstance(v, collections.Callable):
```

Comment:
This and other changes in python code was already present in your python3 branch.